Docs/source/dataanalysis/catalyst.rst

```diff
@@ -0,0 +1,167 @@
+.. _visualization-catalyst:
+
+In situ Visualization with Catalyst 2
+=====================================
+Catalyst 2 (further referred to as just Catalyst) is a lightweight in-situ visualization and analysis framework API developed for simulations and other scientific data producers. It has a lightweight implementation
+(or **stub**) and an SDK to develop custom implementations of Catalyst. ParaView comes with its own implementation (known as **ParaView Catalyst**) for leveraging ParaView's
+visualization and analysis capabilities, which is what this document will focus on.
+
+
+Enabling Catalyst
+-----------------
+In order to use Catalyst with WarpX, you must `build Catalyst 2 <https://catalyst-in-situ.readthedocs.io/en/latest/build_and_install.html>`_ and `build <https://github.com/Kitware/ParaView/blob/master/Documentation/dev/build.md>`__ or `install <https://www.paraview.org/download/>`__ ParaView 5.9+. Afterward, AMReX must be built with ``AMReX_CONDUIT=TRUE``,
+``AMReX_CATALYST=TRUE``, ``Conduit_DIR=/path/to/conduit``, and ``Catalyst_DIR=/path/to/catalyst`` (``/path/to/catalyst`` should be the directory containing ``catalyst-config.cmake``, not the path to the implementation).
```

Comment:
I am on the edge if the install details of how to set CMake hints, besides the options `AMReX_CONDUIT` and `AMReX_CATALYST`.

`/path/to/catalyst`: usually that is the path to the prefix/root of a package install path, if installed correctly:
https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure

`_DIR` variables: correct, but usually we just add the prefixes to the `CMAKE_PREFIX_PATH` environment variable.

I am ok to leave it for now.